web/config-web/etc-apache2-sites-available/000-default.conf

```diff
@@ -1,154 +0,0 @@
-ServerName localhost:8080
```

Comment:
I think we should at least have a basic configuration, as we will need to handle it in the future using Ansible